.github/workflows/changelog.yml

```diff
@@ -0,0 +1,77 @@
+name: changelog
+
+on:
+  pull_request:
+    types: [opened, synchronize, labeled, unlabeled, reopened]
+
+env:
+  # Also change CACHE_VERSION in the other workflows
+  CACHE_VERSION: 8
+  DEFAULT_PYTHON: 3.10
+
+jobs:
+  check-changelog:
+    if: contains(github.event.pull_request.labels.*.name, 'skip news :mute:') != true
```

Comment:
I wonder how this interacts with setting this as a required action. Are skipped actions considered as being successful? Otherwise we might need a first step that checks the if statement and then set the state to successful and exit.

Reply:
I don't really know, I'm no expert in Github actions - so I say we just set the "Skip news" label on this PR and see what happens. 😄

Reply:
Well I tried and that is inconclusive.
https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/troubleshooting-required-status-checks#handling-skipped-but-required-checks also doesn't make me that much wiser.

@Pierre-Sassoulas Could you see the changelog action as a required check and see if the message about missing required checks pops up right now?

Reply:
We don't have any required checks right now so maintainers can take executives decisions in case something is down (or simply merge before any tests ran after fixing a small typo instead of waiting 35mn again). Do we want to make this check required ?

Reply:
(Making it required affects maintainers, not contributors, we're not going to merge PR that do not pass the checks).

Reply:
I think it makes sense to turn it on for this check.

Either way, I'd like to see if the way we currently skip the workflow works with required checks because if not I'd like to make it compatible from the start.

Reply:
This is also going to prevent any other MR to be merged until we merge this one and the check really exists, maybe we could do this in a follow-up to avoid that ?

Reply:
If you ping me just before you do it I'll do a quick check of the result and you can disable it again :)

Reply:
Should be required now :)

Reply:
By requiring being up to date with `main` I can't see if this passes required tests without making a merge commit 😅 

Could you turn that off?

Reply:
I don't think I changed that. Maybe requiring a job require the branch to be up to date ?

Reply:
The button to allow merging `main` from within the Github interface is now also on. Is that also a requirement? That's annoying..

Reply:
<img width="968" alt="Schermafbeelding 2022-07-10 om 23 31 31" src="https://user-images.githubusercontent.com/13665637/178162744-4f85d54c-83c3-43c7-9be2-3871b8072cf7.png">

It's the second option. That should be turned off.

Reply:
Ha thanks @DanielNoord, it reactivate the option automatically when you tick "require status checks"  so you have to deactivate everytime !

Reply:
This seems to work.

<img width="1112" alt="Schermafbeelding 2022-07-11 om 08 26 39" src="https://user-images.githubusercontent.com/13665637/178201800-153b9515-4888-46af-9809-2eb6c8789eca.png">

So we can leave this as is.